### PR TITLE
Use ORCID for curator references

### PR DESCRIPTION
**Pull request by cthoyt**
Related to https://github.com/cthoyt/orcidio/issues/5

This PR turns local provenance information that uses *ad hoc* CURIEs like `CARO:MAH` into using ORCID URIs. Still requires deconvoluting the following initials:

- [ ] SPR
- [ ] gg
- [ ] MA
- [ ] GJ
- [ ] VH
- [ ] DOS-VH (maybe david and someone else?)
- [ ] ds55
- [ ] ds555
- [ ] BrainName (what to do about provenance for this?)

Note from DOS:
> MA = Michael Ashburner
VH = Volker Hartenstein
gg = Gary Grumbling
SPR = Simon Reeve
GJ = Greg Jefferis
BrainName = https://pubmed.ncbi.nlm.nih.gov/24559671/ (Could credit Kei Ito dirtectly)
DOS is me.  ds55 and ds555 might be me (picked up from some account name - IIRC the old editing software did that automatically if you didn’t set it explicitly)

### CODE REVIEW

File: src/ontology/caro-edit.owl [hunk 1]
```diff
@@ -770,7 +771,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/CARO_0001007> <http://purl.obolibrary
 # Class: <http://purl.obolibrary.org/obo/CARO_0001008> (gross anatomical part)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CARO_0001008> "Anatomical structure that is part of a multicellular organism and is at the gross anatomical level, e.g. above the level of a cell.  Included are portions of organism substances such as blood, multi-cell-part structures such as axon tracts, acellular anatomical structures such as hair, and organism subdivisions such as head. Excluded is the whole organism and more granular parts of the organism, such as atoms, molecules, macromolecular complexes and cells. "^^xsd:string)
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/CARO_0001008> "mah"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/CARO_0001008> <https://orcid.org/0000-0001-9114-8737>)
```
Comment (matentzn) [inline src/ontology/caro-edit.owl:774]: There is a very sad situation that MAH are two people, the second one being:

https://dblp.org/pid/33/1638.html

Comment (matentzn) [reply src/ontology/caro-edit.owl:774]: You will have to check with @dosumis on slack (he wont answer here) if we can assume for all CARO ids that MAH is melissa, and not midori (both have curated thousands of classes)

Comment (cthoyt) [reply src/ontology/caro-edit.owl:774]: this is a mega facepalm.

Comment (dosumis) [reply src/ontology/caro-edit.owl:774]: MAH in CARO is all Melissa.

Comment (cthoyt) [reply src/ontology/caro-edit.owl:774]: that's what I assumed since she was on the ontology-level authors list. Thanks for confirming

**Review by matentzn — APPROVED** [review]

**Pull request merged**